Server/mods/deathmatch/logic/CPlayerManager.cpp

```diff
@@ -138,6 +138,19 @@ CPlayer* CPlayerManager::Get(const char* szNick, bool bCaseSensitive)
     return NULL;
 }
 
+CPlayer* CPlayerManager::GetBySerial(const SString& serial) const noexcept
+{
+    for (auto& player : m_Players)
```

Comment:
const?

Reply:
i.e. const auto

also std::string if possible instead of SString

Reply:
Thanks